testcases/OpTestEEH.py

```diff
@@ -82,8 +94,29 @@ def gather_logs(self):
         c.run_command(cmd)
         c.run_command("dmesg")
 
+    def check_eeh_hit(self):
+        c = self.cv_SYSTEM.sys_get_ipmi_console()
+        res = c.run_command("dmesg")
```

Comment:
It would be better to do the grep here, so we don't print full dmesg output over IPMI (which is slow, and unreliable)